packages/contracts/test/utils/web3_wrapper.ts

```diff
@@ -1,5 +1,5 @@
 import { devConstants, env, EnvVars, web3Factory } from '@0xproject/dev-utils';
-import { prependSubprovider } from '@0xproject/subproviders';
+import { prependSubprovider, Web3ProviderEngine } from '@0xproject/subproviders';
```

Comment:
With our new tsconfig settings, the TypeScript compiler requires this type to be explicitly imported.